src/main/java/com/cen4010/bookstore/profileManagement/service/UserServiceImpl.java

```diff
@@ -21,51 +25,57 @@
 public class UserServiceImpl implements UserService {
 
   private final UserRepository userRepository;
+  private final CreditCardRepository creditCardRepository;
 
   @Autowired
-  public UserServiceImpl(UserRepository userRepository) {
+  public UserServiceImpl(UserRepository userRepository,
+      CreditCardRepository creditCardRepository) {
     this.userRepository = userRepository;
+    this.creditCardRepository = creditCardRepository;
   }
 
+  /* ******************************************************************************************** */
   @Override
   public ResponseEntity<UserDto> save(UserDto dto) {
 
     UserEntity entity;
     try {
-      log.info("Saving User Details");
+      log.info("SAVING USER DETAILS");
 
       entity = dto.toEntity();
       entity = userRepository.save(entity);
 
     } catch (DataIntegrityViolationException e) {
-      log.error("Error when saving User Details");
-      throw new DuplicateRecordException("Cannot save duplicate Usernames", dto.getUsername());
+      log.error("ERROR WHILE SAVING USER DETAILS");
+      throw new DuplicateRecordException("CANNOT SAVE DUPLICATE USERS", dto.getUsername());
     }
 
-    log.info("Completed saving User Details");
+    log.info("COMPLETED SAVING USER DETAILS");
 
     return ResponseEntity.status(HttpStatus.OK).body(entity.toDto());
   }
 
+  /* ******************************************************************************************** */
   @Override
-  public ResponseEntity<UserDto> update(UserDto dto, Long userId) {
+  public ResponseEntity<UserDto> update(UserDto dto, UUID userId) {
 
-    log.info("Updating User Details");
+    log.info("UPDATING USER DETAILS");
 
     UserEntity entityExist = userRepository.findById(userId).orElseThrow(() ->
-        new RecordNotFoundException("User Not Found", dto.getUsername()));
+        new RecordNotFoundException("USER NOT FOUND", dto.getUsername()));
 
     entityExist.setPassword(dto.getPassword());
     entityExist.setFullName(dto.getFullName());
 
     entityExist = userRepository.save(entityExist);
 
-    log.info("Completed Updating User Details");
+    log.info("COMPLETED UPDATING USER DETAILS");
 
     return ResponseEntity.status(HttpStatus.OK).body(entityExist.toDto());
 
   }
 
+  /* ******************************************************************************************** */

```

Comment:
There are a lot of these- gonna stop commenting on them

Reply:
Yeah they're "separators" I put in the other night to make things easier since I had a headache. I'll delete them from the master later on with the next pull request